api/types/discoveryconfig/discoveryconfig.go

```diff
@@ -58,7 +63,7 @@ type Spec struct {
 }
 
 // Equal checks if the discovery config is equal to another.
-func (m *DiscoveryConfig) Equal(n *DiscoveryConfig) bool {
```

Comment:
Should we keep Equal and make it call IsEqual to avoid a breaking change? Maybe keep it around for one major or so?

Reply:
I added it 1 release ago so I guess no one is using it.

Reply:
No one in Teleport's codebase, but this is api/. It could be used by someone outside of Teleport.

Reply:
If this never made it out of master then I'm fine with dropping the method. When you said "1 release ago" I figured this had made it into v15.